desktop/src/main/java/bisq/desktop/util/validation/RevolutValidator.java

```diff
@@ -17,10 +17,13 @@
 
 package bisq.desktop.util.validation;
 
-public final class RevolutValidator extends PhoneNumberValidator {
+public final class RevolutValidator extends LengthValidator {
```

Comment:
We have to consider that even the app forces users to set userName now at startup we should not cause bigger problems if the phone number or email is used (via getuserName). Not sure if validator is used in such a potential use case, but not worth to make usage of the account impossible just because of a more strict validator. Also requirements at Revolut might change and our validation would fail then. I think we should not have a close dependency on that.